libp2p/protocols/pubsub/rpc/protobuf.nim

```diff
@@ -288,7 +288,8 @@ proc decodeMessage*(pb: ProtoBuffer): ProtoResult[Message] {.inline.} =
   if ?pb.getField(4, msg.topic):
     trace "decodeMessage: read topic", topic = msg.topic
   else:
-    trace "decodeMessage: topic is missing"
+    error "decodeMessage: topic is required but missing"
```

Comment:
broadly we don't log errors for something that is under the control of a remote peer as this would allow spamming users (who are normally assumed to run on `INF` level) - trace is the right level here (or possibly debug, since it should be uncommon)

Reply:
Oh, alright, will get it updated.